internal/lookoutingester/lookoutdb/insertion_test.go

```diff
@@ -37,10 +37,11 @@ const (
 )
 
 var (
-	baseTime, _     = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:05.000Z")
-	updateTime, _   = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:06.000Z")
-	startTime, _    = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:07.000Z")
-	finishedTime, _ = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:08.000Z")
+	baseTime, _      = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:05.000Z")
+	updateTime, _    = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:06.000Z")
+	startTime, _     = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:07.000Z")
+	finishedTime, _  = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:08.000Z")
+	preemptedTime, _ = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:08.000Z")
```

Comment:
you might want to make the time here `2022-03-01T15:04:09.000Z` so it's different from `finishedTime`?

Reply:
will do